backend/lib/endpoints/posts.js

```diff
@@ -233,6 +233,9 @@ async function routes(app) {
             title: true,
             types: true,
             visibility: true,
+            createdAt: true,
+            elapsedTimeText: true,
```

Comment:
`elapsedTimeText` is a virtual
> aggregate operation is performed server-side, where any client-side Mongoose concepts like virtuals do not exist.

so it will be undefined, so you can remove it along with the new `elapsedTimeText` virtual in the Post model.